src/sagas/giphy.js

```diff
@@ -0,0 +1,96 @@
+import { call, takeEvery, put,takeLatest  } from 'redux-saga/effects';
+import {getOneGif, getTrendingGifs, searchGif, addLikeGif, getLikedGifs, removeLikeGif, getLikedGifsSearch } from '../api/giphy'
+import gifs from '../actions/gifs'
+
+
+
+export function* loadTrendingGifs(){
+    try{
+        const response = yield call(getTrendingGifs)
+        
+        //poner aqui el acion para mostar gifs
```

Comment:
These comments should be removed, I would recommed using `TODO: action still pending` in order to find them and remove them easily when the feature is implemented.